site/content/contribute/server/developer-workflow.md

```diff
@@ -11,59 +11,59 @@ Join the [Developers community channel](https://community.mattermost.com/core/ch
-2. On your fork, create a branch `MM-####` where #### is the ticket number if it is a [Jira](https://mattermost.atlassian.net) ticket, or `GH-####` if it is a GitHub Issue without a Jira ticket.
-3. Make the code changes required to complete your ticket, making sure to write or modify unit tests where appropriate. Make sure to use [testify](https://github.com/stretchr/testify) for new tests.
+    * [./api4/](https://github.com/mattermost/mattermost-server/tree/master/api4) holds all API and application related code.
+    * [./model/](https://github.com/mattermost/mattermost-server/tree/master/model) holds all data model definitions and the Go driver.
+    * [./store/](https://github.com/mattermost/mattermost-server/tree/master/store) holds all database querying code.
+    * [./utils/](https://github.com/mattermost/mattermost-server/tree/master/utils) holds all utilities, such as the mail utility.
+    * [./i18n/](https://github.com/mattermost/mattermost-server/tree/master/i18n) holds all localization files for the server.
+2. On your fork, create a branch `MM-####` where #### is the ticket number if it's a [Jira](https://mattermost.atlassian.net) ticket, or `GH-####` if it's a GitHub Issue without a Jira ticket.
+3. Make the code changes required to complete your ticket, ensuring that unit tests are written or modified  where appropriate. Please use [testify](https://github.com/stretchr/testify) for new tests.
 4. To test your changes, run `make run-server` from the root directory of the server respository. This will start up the server at `http://localhost:8065`. To get changes to the server it must be restarted with `make restart-server`. If you want to test with the web app, you may also run `make run` which will start the server and a watcher for changes to the web app.
 5. Once everything works to meet the ticket requirements, stop Mattermost by running `make stop` in the server repository, then run `make check-style` to check your syntax.
-6. Run the tests. You have a couple alternatives:
-     * Run `make test` to run all the tests in the project, but that may take a long time and provide very little feedback while it is running.
+6. Run the tests using one or more of the following options:
+     * Run `make test` to run all the tests in the project. This may take a long time and provides very little feedback while it's running.
      * Run individual tests by name executing `go test -run "TestName"`.
      * Run all the tests in a package where changes were made executing `go test app`.
      * Create a draft PR with your changes and let our CI servers run the tests for you.
-7. Commit your changes, push your branch and [create a pull request](https://developers.mattermost.com/contribute/getting-started/contribution-checklist/).
+7. Commit your changes, push your branch, and [create a pull request](https://jeffschering.github.io/mmdocs/upgrade/developer/contribution-guide.html#preparing-a-pull-request/).
```

Comment:
What is https://jeffschering.github.io/mmdocs/upgrade`? Your personal fork? Why would you use this over https://developers.mattermost.com/contribute/getting-started/contribution-checklist/?

Reply:
Good question, and an error on my part. I do think Jeff's content is more detailed about what a pull request entails but agree that using that URL is not ideal. 

It's a good case for updating the Mattermost contribution checklist though.